entity-framework/core/cli/dbcontext-creation.md

```diff
@@ -39,11 +39,11 @@ You can also tell the tools how to create your DbContext by implementing the `ID
 > This is fixed in EFCore 5.0 and any additional design-time arguments
 > are passed into the application through that parameter.
 
-A design-time factory can be especially useful if you need to configure the DbContext differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
+A design-time factory can be especially useful if you need to configure the `DbContext` differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
 
 ## Args
 
-Both IDesignTimeDbContextFactory.CreateDbContext and Program.CreateHostBuilder accept command line arguments.
+Both `IDesignTimeDbContextFactory.CreateDbContext` and `Program.CreateHostBuilder` accept command line arguments.
```

Comment:
As per https://github.com/dotnet/EntityFramework.Docs/issues/3064#issuecomment-771760832, these should use <xref> to reference the API docs.

Reply:
fixed